Resources/Prototypes/XenoArch/effects.yml

```diff
@@ -0,0 +1,33 @@
+- type: entity
```

Comment:
need to move all old effects for initial merge. But make them parametrized to actually use them in custom graph creation logic later!